### PR TITLE
Add flag to disable sort groups by name, by default sorting is enabled

### PR DESCRIPTION
**Pull request by rafalwojcik**
In request to issue #26 and for my usage :)

### CODE REVIEW

File: bin/synx [hunk 1]
```diff
@@ -9,6 +9,7 @@ Clamp do
   option ["--prune", "-p"], :flag, "remove source files and image resources that are not referenced by the the xcode project"
   option "--no-color", :flag, "removes all color from the output"
   option "--no-default-exclusions", :flag, "doesn't use the default exclusions of /Libraries, /Frameworks, and /Products"
+  option "--disable-sort-by-name", :flag, "disable sorting groups by name"
```
Comment (eliperkins) [inline bin/synx]: What do you think about naming this `--no-sort-by-name` for consistency with the other options?

Comment (rafalwojcik) [inline bin/synx]: Ok i can change that :)
:+1:

**Comment by eliperkins** [comment]
Awesome. Looks good to me! Can you do me a favor and rebase this down into one commit, just to keep git history clean?

:green_apple:

**Comment by rafalwojcik** [comment]
Sure i just writing tests now :) So give me one more second.

**Comment by rafalwojcik** [comment]
Done ;)

File: lib/synx/project.rb [hunk 2]
```diff
@@ -49,6 +49,12 @@ def set_options(options)
       end
 
       self.group_exclusions |= options[:group_exclusions] if options[:group_exclusions]
+      
+      if options[:no_sort_by_name]
```
Comment (marklarr) [inline lib/synx/project.rb]: Could this be simplified to `self.sort_by_name = !!options[:no_sort_by_name]`?

**Comment by marklarr** [comment]
This looks good. One question, though -- the title of the PR says `by default sorting is disabled`, but, looking at the code, it looks like it's enabled by default, and you opt-out with the flag `--no-sort-by-name`?

**Comment by rafalwojcik** [comment]
@marklarr Yep sorry for inconsistent, first version was with disabled, but i change it and forget to edit PR title :)

File: lib/synx/project.rb (hunk 2 above)
Comment (rafalwojcik) [inline lib/synx/project.rb]: :+1: 
Of course it can be like this. I'm iOS developer so not so much experience in Ruby :)

Comment (rafalwojcik) [inline lib/synx/project.rb]: Done :)

**Comment by marklarr** [comment]
Sweet! Looks good. I'll release this tonight.

**Comment by marklarr** [comment]
`0.2.0` released

**Pull request merged**